test_schemas/en/test_generate_date_range.json

```diff
@@ -0,0 +1,223 @@
+                                "title": {
+                                    "text": "This is a formatted date range with some literal dates in a different month and same year: {formatted_date_range}",
+                                    "placeholders": [
+                                        {
+                                            "placeholder": "formatted_date_range",
+                                            "transforms": [
+                                                {
+                                                    "transform": "format_date_range_pair",
+                                                    "arguments": {
+                                                        "date_range": {
+                                                            "value": ["2019-05-01", "2019-06-01"]
+                                                        }
+                                                    }
+                                                }
+                                            ]
+                                        }
+                                    ]
+                                },
+                                "type": "General",
+                                "answers": [
+                                    {
+                                        "id": "date-range-month-answer",
+                                        "label": "How many days did you work during this period?",
+                                        "mandatory": false,
+                                        "type": "Number"
+                                    }
+                                ]
+                            }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "range-year-question-block",
+                            "question": {
+                                "id": "range-year-question",
+                                "title": {
+                                    "text": "This is a formatted date range with some literal dates in a different year: {formatted_date_range}",
+                                    "placeholders": [
+                                        {
+                                            "placeholder": "formatted_date_range",
+                                            "transforms": [
+                                                {
+                                                    "transform": "format_date_range_pair",
+                                                    "arguments": {
+                                                        "date_range": {
+                                                            "value": ["2019-05-01", "2020-05-01"]
+                                                        }
+                                                    }
+                                                }
+                                            ]
```

Comment:
For this, if we should drive it based on 2 dynamic inputs, could be 2 answers or you could use metadata dates we have in scope.
```json
{
    "source": "metadata",
    "identifier": "ref_p_start_date"
}
```
There is also an `ref_p_end_date`. To get more info on how it is used in a schema, have a look at `test_date_validation_yyyy_combined.json`.